am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -17,4 +18,18 @@ public void createResourceAccess(String resourceId, String accessorId) {
         jdbi.useExtension(AccessManagementRepository.class,
             dao -> dao.createAccessManagementRecord(resourceId, accessorId));
     }
+
+    /**
+     * Returns `resourceJSON` when row with userId (accessorId) and resourceId is in Db, otherwise null
+     * @param userId (accessorId)
+     * @param resourceId
+     * @param resourceJSON
+     * @return resourceJSON or null
+     */
+    public JsonNode filterResource(String userId, String resourceId, JsonNode resourceJSON) {
+        boolean hasAccess = jdbi.withExtension(AccessManagementRepository.class,
+                dao -> dao.explicateAccessExist(userId, resourceId));
```

Comment:
is it explicate access on purpose or should it be explicit?

Reply:
should be explicit